src/Components/js/Navigation.jsx

```diff
@@ -0,0 +1,24 @@
+function Navigation() {
+  return (
+    <>
+      <div className="navigation-container">
```

Comment:
So the <nav> element is actually more appropriate here than using a div to wrap everywhere. You can remove the div and keep the nav